java/rocksjni/cplusplus_to_java_convert.h

```diff
@@ -0,0 +1,35 @@
+#ifndef _CPLUSPLUS_TO_JAVA_CONVERT_H
+#define _CPLUSPLUS_TO_JAVA_CONVERT_H
```

Comment:
Is there a reason we don't use
```
#pragma once
```
as other header files?

Reply:
Thanks @siying I have now switched it to use that